src/site/utilities/featureFlags.js

```diff
@@ -112,43 +109,12 @@ const flagsByBuildtype = {
 };
 
 // Exported feature flag state, which can be used in code as needed
-const enabledFeatureFlags = Object.values(featureFlags).reduce((acc, next) => {
+const cmsFeatureFlags = Object.values(featureFlags).reduce((acc, next) => {
   acc[next] = flagsByBuildtype[global.buildtype].includes(next);
   return acc;
 }, {});
 
-const applyFeatureFlags = (moduleToFlag, flagToUse = null) => {
```

Comment:
Turns out this isn't used anywhere anymore. Down with vestigial code! :fist_raised: